catalog/spatial/spatial-app/src/main/resources/features.xml

```diff
@@ -121,24 +121,30 @@
         </bundle>
     </feature>
 
+    <feature name="spatial-geocoding-endpoint" version="${project.version}"
+             description="Spatial GeoCoding Endpoint.">
+        <bundle>mvn:org.codice.ddf.spatial/spatial-geocoding-api/${project.version}</bundle>
+        <bundle>mvn:org.codice.ddf.spatial/spatial-geocoding-geocoder/${project.version}</bundle>
```

Comment:
✏️ Does it may make more sense to change `spatial-geocoding-geocoder` to `spatial-geocoding-impl`?

Reply:
I didn't want to change the module name because it has other classes in it than just the impl